docs/data-collection/tear-down.md

```diff
@@ -66,29 +62,34 @@
     - [ ] Remove the back padding elements and put them back in their designated storage.
     - [ ] Reinstall the spine coil.
     - [ ] Plug back the head coil if you know the next exam will require that specific coil, or simply put it away with the other (head) coils on the shelf next to the scanner.
-    - [ ] Put the bed back in place = push the "home" button on the scanner
+    - [ ] Put the bed back in place = push the home (:fontawesome-solid-house:) button on the scanner
     - [ ] Put the wooden stopper of the main access tube back on the tube.
     - [ ] Everything that is removed for the experiment MUST be put back in place at the end of the experiment, i.e. position of the bed, coil, emergency button, ears padding.
     - [ ] Take a glove, on the right there is some cleaning napkins. Use them to clean the bed.
-    - [ ] Exit and close the external door.
+    - [ ] Take the ECG electrodes, the RB, and the plexiglas outside to the control room.
+    - [ ] Close the external door.
 
-## AFTER SCAN, outside scanner room
+## AFTER SCAN, outside the scanner room
 
-- [ ] Switch off the projector.
+- [ ] Help the person inside the scanning room to extract the cables and tubes.
+- [ ] Carefully roll the fiber and power cable of the ET, and place them in the rolling table of the ET PC Tower.
+- [ ] Carefully roll RB and the GA tubes and put them back in the GA and BIOPAC boxes.
+- [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it and put it back into its cover.
 
+  ![cover-mri-compatible-lens](../assets/images/cover-mri-compatible-lens.png "Cover MRI compatible lens")
+
+- [ ] Put the cover, the ET base back in the fMRI box, **being extremely careful to not crush the mirror**.
 - [ ] **Retrieve ET recordings** (from {{ secrets.hosts.psychopy | default("███") }}):
     - [ ] Insert a USB key into *{{ secrets.hosts.psychopy | default("███") }}* and save the experiment from AcqKnowledge.
     - [ ] Upload to a pre-designated drop-box (e.g., using Dropbox)
-- [ ] Press <span class="keypress">Ctrl</span>+<span class="keypress">Alt</span>+<span class="keypress">Q</span> on the ET's computer to exit the EyeLink 1000 Plus Host PC application and click on the <span class="keypress">Shutdown</span> button from the **File Manager** toolbar.
+- [ ] Press <span class="keypress">Ctrl</span>+<span class="keypress">Alt</span>+<span class="keypress">Q</span> on the ET PC Tower to exit the EyeLink 1000 Plus Host PC application and click on the <span class="keypress">Shutdown</span> button from the **File Manager** toolbar.
```

Comment:
@esavary Have you tested this on our setup? From what I remember, when I exited the Eyelink application, I was left with a black screen, so "click on the Shutdown button from the File Manager toolbar" would not have been possible.

Reply:
I'll double check on the Eyetracker. I had a DOS like screen last time I tried to exit the application.

Reply:
Since this comment has not been resolved within this PR, I just opened #166 to follow up.